geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/accumulo.scala

```diff
@@ -53,6 +53,9 @@ package object accumulo {
       val QUERY_EXACT_COUNT    = PropAndDefault("geomesa.force.count", "false")
       val QUERY_COST_TYPE      = PropAndDefault("geomesa.query.cost.type", null)
       val QUERY_TIMEOUT_MILLIS = PropAndDefault("geomesa.query.timeout.millis", null) // default is no timeout
+      // rough upper limit on the number of accumulo ranges we will generate per query
+      val SCAN_RANGES_TARGET   = PropAndDefault("geomesa.scan.ranges.target", "2000")
```

Comment:
yeah, maybe it should be in the data store config - it's kind of a cloud-specific thing I think, but you could have datastores pointing to multiple clouds in a single jvm. 2k didn't really come from anywhere - that was our target for the old recursive range method, it's just that we usually went over by a lot. Think more would be better? i was trying to mitigate the multi-week z3 bog-down (this is ranges per-week for z3)